invokeai/backend/model_manager/probe.py

```diff
@@ -643,17 +646,19 @@ def _guess_name(self) -> str:
         return name
 
 
-class TextualInversionFolderProbe(FolderProbeBase):
-    def get_format(self) -> ModelFormat:
-        return ModelFormat.EmbeddingFolder
-
-    def get_base_type(self) -> BaseModelType:
-        path = self.model_path / "learned_embeds.bin"
-        if not path.exists():
+class TextualInversionFolderProbe(TextualInversionCheckpointProbe):
+    def __init__(self, model_path: Path):
+        files = os.scandir(model_path)
+        files = [
+            Path(f.path)
+            for f in files
+            if f.is_file() and f.name.endswith((".ckpt", ".pt", ".pth", ".bin", ".safetensors"))
+        ]
+        if len(files) != 1:
             raise InvalidModelConfigException(
-                f"{self.model_path.as_posix()} does not contain expected 'learned_embeds.bin' file"
+                f"Unable to determine base type for {model_path}: expected exactly one valid model file, found {[f.name for f in files]}."
             )
-        return TextualInversionCheckpointProbe(path).get_base_type()
+        super().__init__(files.pop())
```

Comment:
What if the first model file we find isn't the correct one?

I've seen a few TI models hosted on diffusers that have multiple checkpoints in them, presumably representing different stages of trainign.

Reply:
> What if the first model file we find isn't the correct one?

Nevermind that, I misread the length check. The other issue still stands.

Reply:
I don't believe any of the checkpoints are actually used for inference but I could be wrong. This only scans the root directory of the folder so as long as there's only 1 model there it loads it fine. It does move it out of the folder though.

Reply:
What I mean is that this logic would appear to fail probing TIs that have multiple model files in the directory.

Some look like this (struggling to find an example, filenames probably aren't accurate):
- `embedding.pt`
- `checkpoint-001.pt`
- `checkpoint-002.pt`

This is a valid TI and the model loader knows how to load it, but I believe the logic in this PR would see the multiple model files and consider it an invalid model.